patterns/dxp/aspect_oriented_programming.md

```diff
@@ -0,0 +1,53 @@
+# Aspect Oriented Programming guide of use
+
+> **NOTE:** This plugin was originally in metal-plugins but now lives in portal so we can provide a better developing experience.
```

Comment:
I've been calling it "liferay-portal" in the other docs to match the repo name on GitHub, and making it a link.

Reply:
liferay-portal also helps distinguish from react portals